ThirdPartyAdapters/vungle/CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 ## Vungle Android Mediation Adapter Changelog
 
+#### Version 6.8.0.0
+- Verified compatibility with Vungle SDK 6.8.0.
```

Comment:
@jiancchen can we please add following:
- Android 11 Support (Since 6.7.1)
- Removed Flex Feed and Flex View formats

As per MattB as there might be some marketing push with ios, so we want to again highlight Android 11 support, as 6.7.1 didnt get any marketing push